README.md

```diff
@@ -103,7 +103,7 @@ cd sushi-fusion
 npm i && npm start
 ```
 
-Then open [http://localhost:3000](){:target="\_blank"} to see the app.
+Then open [http://localhost:3000](http://localhost:3000) to see the app.
```

Comment:
The task statement is not clear but,
I think this should work.
Please let me know if that is ok.

Reply:
Please confirm with Ana Pirris,I am also a bit confused